src/Alert/Alert.tsx

```diff
@@ -0,0 +1,222 @@
+        color:
+          theme === "light"
+            ? colors.black.base
+            : theme === "dark"
+            ? colors.white
+            : assertUnreachable(theme),
+        boxShadow: `0 4px 8px 0 rgba(0, 0, 0, .04)`,
+        borderStyle: "solid",
+        borderRadius: 4,
+        borderWidth: theme === "light" ? 1 : 0,
+        borderColor: colors.silver.dark,
+        padding: 15,
+      }}
+    >
+      <div
+        css={{
+          marginBottom: extended ? 14 : 6,
+          overflow: "hidden",
+          display: "flex",
+        }}
+      >
+        <ClassNames>
+          {({ css, cx }) => {
+            const headingProps = {
+              className: cx(
+                css({
+                  fontWeight: 600,
+                  marginBottom: 0,
+                  marginTop: 0,
+                  width: "100%",
+                  display: "flex",
+                  color: getOffsetInPalette(
+                    2,
+                    theme === "light" ? "darker" : "lighter",
+                    color
+                  ),
+                  ...base.base,
+                })
+              ),
+              children: (
+                <Fragment>
+                  <ClassNames>
+                    {({ css, cx }) =>
+                      React.cloneElement(icon, {
+                        className: classnames(
+                          icon.props.className,
+                          cx(
+                            css({
+                              width: 20,
+                              height: 20,
+                              color,
+                              marginRight: 13,
+                            })
+                          )
+                        ),
+                      })
+                    }
+                  </ClassNames>
+                  {heading}
+                </Fragment>
+              ),
+            };
+
+            return React.isValidElement(headingAs)
+              ? React.cloneElement(headingAs, {
+                  ...headingProps,
+                  className: classnames(
+                    headingProps.className,
+                    headingAs.props.className
+                  ),
+                })
+              : React.createElement(headingAs, headingProps);
+          }}
+        </ClassNames>
+        <IconClose
+          onClick={onClose}
+          css={{
+            color: colors.grey.lighter,
+            cursor: "pointer",
+            width: 10,
+            height: 10,
+          }}
+        />
+      </div>
+
+      {extended && (
+        <hr
+          css={{
+            height: 1,
+            borderWidth: 0,
+            backgroundColor:
+              theme === "dark" ? colors.grey.dark : colors.silver.dark,
+            marginTop: 14,
+            marginBottom: 14,
+          }}
+        />
+      )}
+      <div css={{ marginLeft: extended ? 0 : 33 }}>
+        <div
+          css={{
+            fontSize: 13,
```

Comment:
minor: Use Space Kit typography for this

https://github.com/apollographql/space-kit/blob/b3fa01c71208ae0e1f0e15b32e85cf846b80a142/src/typography/base.ts#L37-L41